scipioninstaller/installer.py

```diff
@@ -162,12 +173,12 @@ def getInstallationCmd(scipionHome, dev, args):
             cmd += cmdfy("python -m scipion installb xmippDev -j %d" % nProcess)
 
     else:
-        cmd = cmdfy("pip install scipion-pyworkflow")
+        cmd += cmdfy("pip install scipion-pyworkflow")
         cmd += cmdfy("pip install scipion-app")
```

Comment:
I would add here
```
if not noXmipp:
    cmd += cmdfy("pip install scipion-em-xmipp")
```
this should install also the binaries from xmippSrc.

Reply:
Not true, binaries would be installed if `scipion-em-xmipp` is installed via `installp`